riscv/src/riscv_asm.lalrpop

```diff
@@ -67,9 +67,17 @@ Directive: Statement = {
 }
 
 Instruction: Statement = {
+    ".insn" <InsnSyntax>,
```

Comment:
At some point, I would like this grammar to be generic for all architectures.
In that case, it might be better to keep this an assembler directive instead of an instruction, and thus do the transformation from directive to instruction after the parser. Would that be difficult to do?

Reply:
Oh sorry, I forgot about that. I'll change this.

Reply:
Sorry, I couldn't do it.

Reply:
I mean it would make the following variant ambiguous (is the + unary or binary?):
```
.insn i +15, 0, zero, zero, 16
```